qemu/tests/qemu_guest_agent.py

```diff
@@ -2804,6 +2830,7 @@ def wrap_windows_cmd(cmd):
             if not utils_disk.update_windows_disk_attributes(session,
                                                              disk_index):
                 test.fail("Can't online disk with fsthaw")
+        self.gagent_setsebool_value('off', params, self.vm)
```

Comment:
Should move to Line 2828 since it is Linux only. And I guess you need a `try...finally` to make sure the value is change back even when the test case failed.